src/main/java/gov/ca/emsa/controller/DirectoryController.java

```diff
@@ -36,18 +36,6 @@ public DirectoryController() {
 		jsonMapper = new ObjectMapper();
 	}
 	
-	@RequestMapping(value= "/mock/Organization", method = RequestMethod.GET, produces="application/json; charset=utf-8")
```

Comment:
This was actually supposed to be here. There is a CTEN url for Organization that we aren't using right now but i wanted to make the mock version just in case we find we need it. This is in addition to the Location and Endpoint URLs that we are using.